tests/test_c_lexer.py

```diff
@@ -448,11 +448,11 @@ def test_char_constants(self):
         self.assertLexerError("'", ERR_UNMATCHED_QUOTE)
         self.assertLexerError("'b\n", ERR_UNMATCHED_QUOTE)
         self.assertLexerError("'\\xaa\n'", ERR_UNMATCHED_QUOTE)
-
-        self.assertLexerError(r"'\12a'", ERR_INVALID_CCONST)
```

Comment:
Not sure why you're modifying these tests, without adding any new tests

Reply:
Added new tests.